plugins/woocommerce/includes/admin/reports/class-wc-report-customer-list.php

```diff
@@ -53,7 +53,7 @@ public function output_report() {
 		if ( ! empty( $_GET['link_orders'] ) && wp_verify_nonce( $_REQUEST['_wpnonce'], 'link_orders' ) ) { // phpcs:ignore WordPress.Security.ValidatedSanitizedInput
 			$linked = wc_update_new_customer_past_orders( absint( $_GET['link_orders'] ) );
 			/* translators: single or plural number of orders */
-			echo '<div class="updated"><p>' . sprintf( esc_html( _n( '%s previous order linked', '%s previous orders linked', $linked, 'woocommerce' ), $linked ) ) . '</p></div>';
+			echo '<div class="updated"><p>' . sprintf( esc_html( _n( '%s previous order linked', '%s previous orders linked', $linked, 'woocommerce' ) ), $linked ) . '</p></div>';
```

Comment:
Could we add an `absint()` around `$linked` to the two occurrences on this line? Just to be safe (and make PHPCS happy).